api/specs.proto

```diff
@@ -19,39 +19,39 @@ import "google/protobuf/wrappers.proto";
 // component or a different representation to reflect allocation or resolution.
 
 message NodeSpec {
-	Annotations annotations = 1 [(gogoproto.nullable) = false];
+  Annotations annotations = 1 [(gogoproto.nullable) = false];
```

Comment:
Please don't have opinions about white space in a PR that removes a single field. :P

Reply:
whoops

Reply:
i was getting "mixed indent" warnings so i did `gg=G` in vim.